docs/index.md

```diff
@@ -64,23 +63,27 @@ Pelorus works by integrating with your application platforms, developer tools, a
 
 Currently, Pelorus functionality can capture proven metrics that measure Software Delivery Performance -- a significant outcome that IT organizations aim to deliver.
 
-Pelorus is a Grafana dashboard that can easily be deployed to an OpenShift cluster, and provides an organizational-level view of the [four critical measures of software delivery performance](https://blog.openshift.com/exploring-a-metrics-driven-approach-to-transformation/).
+Pelorus is an OpenShift Operator that can easily be deployed to an OpenShift cluster, and provides an organizational-level view of the [four critical measures of software delivery performance](https://blog.openshift.com/exploring-a-metrics-driven-approach-to-transformation/).
```

Comment:
Should we update this?

Reply:
I will undo these changes and we will make them in https://github.com/konveyor/pelorus/pull/770